carla_controller/weather.py

```diff
@@ -0,0 +1,68 @@
+import carla
+
+from random import choices, uniform
+
+
+def random_weather():
```

Comment:
Ah, didn't realize weather was a function/file you create.  This makes more sense now.

Reply:
Yeah but I'll remove this for the premade static defaults you pointed out.